sdk/storage/Azure.Storage.Common/src/Shared/UriExtensions.cs

```diff
@@ -113,7 +113,7 @@ public static string GetAccountNameFromDomain(string host, string serviceSubDoma
         /// <returns>True if using IP Endpoint style.</returns>
         public static bool IsHostIPEndPointStyle(this Uri uri) =>
             (!string.IsNullOrEmpty(uri.Host) &&
-            uri.Host.IndexOf(".", StringComparison.InvariantCulture) >= 0 &&
+            uri.Host.Contains(".") &&
```

Comment:
Why was this change necessary?  Can we pass the invariant culture into `Contains` at least?

Reply:
Netstandard2.1 does not like `IndexOf()` and warns. I can try and put invariant back in there.